L1Trigger/L1TTrackMatch/interface/L1TkHTMissEmulatorProducer.h

```diff
@@ -47,7 +47,7 @@ namespace l1tmhtemu {
     kUnassignedMSB = kUnassignedLSB + kUnassignedSize - 1
   };
 
-  const float kMaxMHT{4096};  // 4 TeV
+  const float kMaxMHT{2048};  // 4 TeV
```

Comment:
Still valid this `4 TeV`?

Reply:
@Chriisbrown Can you answer this question?

Reply:
The comment should be updated to 2 TeV